src/pages/Calendar/Caption/caption.style.ts

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components/native";
+import { ColorScheme } from "@styles/globalStyles";
+import { FontScheme } from "@styles/globalStyles";
+
+export const Sc = {
+  Captions: styled.View`
+    flex-direction: row;
+    gap: 96px;
+  `,
+
+  LeftCaptions: styled.View`
+    flex-direction: column;
+  `,
+
+  RightCaptions: styled.View`
+    flex-direction: column;
+  `,
```

Comment:
These 2 components aren't actually changing anything. In react native,  all View components have `flex-direction: column` as default. So you could rename it to something else and replace the LeftCaptions and RightCaptions with it. For example (you can name it whatever you wish)

```js
  Division: styled.View``,
  // And yes, you can leave it empty
```